io.github.amourspirit.LibrePythonista_PyEditor.yml

```diff
@@ -0,0 +1,62 @@
+app-id: io.github.amourspirit.LibrePythonista_PyEditor
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: cell_edit
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
```

Comment:
why does a console app need these?

Reply:
It is not a console app. I used the [AppStream MetaInfo Creator](https://www.freedesktop.org/software/appstream/metainfocreator) to create the metadata. It seemed to be the only template that was close to my app. I have now removed the console type.

This app installs my [librepythonista-python-editor](https://pypi.org/project/librepythonista-python-editor/) which is a [PyWebveiw](https://pywebview.flowrl.com/) based Python project.

```
 - --socket=wayland
 - --socket=fallback-x11
 - --share=ipc
 ```
Is required for the PyWebview (webview) window to display.